src/Lykke.Service.Assets/Startup.cs

```diff
@@ -151,7 +149,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
 
                 app.UseMvc();
                 app.UseSwagger();
```

Comment:
Call as in the template https://github.com/LykkeCity/lykke.dotnettemplates/blob/master/Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Startup.cs#L89